annotation/functions.scm

```diff
@@ -23,10 +23,9 @@
     #:use-module (opencog exec)
     #:use-module (opencog bioscience)
     #:use-module (annotation util)
-;    #:use-module (rnrs base)
     #:use-module (srfi srfi-1)
     #:use-module (ice-9 match)
-    #:export (find-ggi find-ppi find-output-interactions)
+    #:export (find-ggi find-ppi find-output-interactions find-go-plus)
```

Comment:
The fist three functions `find-ggi`` find-ppi` `find-output-interactions` are not defined here